SPEC.md

```diff
@@ -15,6 +15,10 @@ The core goals of the specification include:
 * Using common technologies for cryptography, archiving, compression and transport
 * Using the DNS namespace to name and discover images
 
+### Requirements
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](http://tools.ietf.org/html/rfc2119).
```

Comment:
I thought of the spec to be a single logical document split into multiple files, but yes, telling this explicitly seems good. Changed.